src/test/java/study/racingcar/CarTest.java

```diff
@@ -0,0 +1,31 @@
+package study.racingcar;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CarTest {
+    Car car;
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/java-racingcar/pull/568#discussion_r396047225

```suggestion
    private Car car;
```